examples/todo/rust/main.rs

```diff
@@ -100,11 +107,66 @@ pub fn main() {
 #[cfg(target_os = "android")]
 #[no_mangle]
 fn android_main(app: i_slint_backend_android_activity::AndroidApp) {
+    use i_slint_backend_android_activity::android_activity::{MainEvent, PollEvent};
     slint::platform::set_platform(Box::new(
         i_slint_backend_android_activity::AndroidPlatform::new_with_event_listener(app, |event| {
-            eprintln!("Got event: {event:?}")
+            eprintln!("Got event: {event:?}");
```

Comment:
I still think that eprintln!() shouldn't be there in our example :)

Reply:
I think is useful so user can see what event to expect.

But I'll remove it at a later point then.